appinventor/docs/html/reference/blocks/control.html

```diff
@@ -168,6 +168,12 @@ <h3 id="foreach">for each item in list</h3>
 
 <p>Runs the blocks in the do section for each item in the list. Use the given variable name, <code class="variable block highlighter-rouge">item</code>, to refer to the current list item. You can change the name <code class="variable block highlighter-rouge">item</code> to something else if you wish.</p>
 
+<h3 id="foreachdict">for each key with value in dictionary</h3>
+
+<p><img src="images/control/foreachdict.png" alt="" /></p>
+
+<p>Runs the blocks in the do section for each key-value entry in the dictionary. Use the given variables, <code class="variable block highlighter-rouge">key</code> and <code class="variable block highlighter-rouge">value</code>, to refer to the key and value of the current dictionary entry. You can change the names <code class="variable block highlighter-rouge">key</code> and <code class="variable block highlighter-rouge">value</code> to something else if you wish. </p>
```

Comment:
When I build this PR there's a whitespace difference at the end of this paragraph.